src/client/components/EditUserForm/CollaboratorPermissions/CollaboratorPermissions.scss

```diff
@@ -0,0 +1,20 @@
+@import 'src/client/style/partials';
+
+.edit-user__form-item-permissions {
+  height: auto;
+}
+
+.edit-user__form-field-premissions {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 12px;
+  font-size: 12px;
```

Comment:
do we have config for these? `spacing-x`?

Reply:
we do, but not sure why they sometimes get ignored

Reply:
we don't

<img width="290" alt="Schermata 2022-10-03 alle 10 31 13" src="https://user-images.githubusercontent.com/10047945/193533933-1abb395c-39d4-4c63-93a5-98682701176f.png">

Reply:
let's open another issue about it because it affects the whole user form